frontend-shared/babel.config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "presets": [
+    [
+      "@babel/env", {
```

Comment:
This is copied from the client, but it might be best to use a single source of truth?